pkg/usage/ping.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2020 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package usage
+
+import (
+	"time"
+
+	"github.com/openebs/cstor-csi/pkg/env"
+)
+
+const (
+	// defaultPingPeriod sets the default ping heartbeat interval
+	defaultPingPeriod time.Duration = 24 * time.Hour
+	// minimumPingPeriod sets the minimum possible configurable
+	// heartbeat period, if a value lower than this will be set, the
+	// defaultPingPeriod will be used
+	minimumPingPeriod time.Duration = 1 * time.Hour
+)
+
+// PingCheck sends ping events to Google Analytics
+func PingCheck() {
```

Comment:
This function is not called from anywhere

Reply:
I think its there for further future enhancements, cc @vharsh